include/clap/ext/draft/scratch-memory.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "../../plugin.h"
+
+// This extension lets the plugin request "scratch" memory
+// from the host. Scratch memory can be accessed during the
+// `process()` callback, but is not persistent between callbacks.
```

Comment:
It'd be great to have a paragraph that explain what are the benefits of using this extension.
I suppose that you were thinking of sharing the scratch buffer between multiple plugin instances?